From imrehg@gmail.com  Fri Oct  9 10:32:19 2009
From: Gergely Imreh <imrehg@gmail.com>
Date: Tue, 15 Sep 2009 16:03:31 +0800
Subject: USB: usbtmc: fix timeout increase
To: Greg KH <greg@kroah.com>
Cc: "linux-usb@vger.kernel.org" <linux-usb@vger.kernel.org>
Message-ID: <371b60070909150103t2e2058f5ua819ab3300d47144@mail.gmail.com>

From: Gergely Imreh <imrehg@gmail.com>

The current 10ms timeout is too short for some normal USBTMC device
operation, increase it to a value which was tested with previously
affected Tektronix oscilloscopes.

Signed-off-by: Gergely Imreh <imrehg@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/class/usbtmc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/class/usbtmc.c
+++ b/drivers/usb/class/usbtmc.c
@@ -39,7 +39,7 @@
 #define USBTMC_SIZE_IOBUFFER	2048
 
 /* Default USB timeout (in milliseconds) */
-#define USBTMC_TIMEOUT		10
+#define USBTMC_TIMEOUT		5000
 
 /*
  * Maximum number of read cycles to empty bulk in endpoint during CLEAR and
